demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -962,15 +962,15 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
       document.getElementById('audio-preview').parentElement.style.visibility = 'hidden';
       return;
     }
-    const data = new Float32Array(analyserNode.fftSize);
+    const data = new Uint8Array(analyserNode.fftSize);
     let frameIndex = 0;
     this.analyserNodeCallback = () => {
       if (frameIndex === 0) {
-        analyserNode.getFloatTimeDomainData(data);
+        analyserNode.getByteTimeDomainData(data);
         const lowest = 0.01;
         let max = lowest;
         for (const f of data) {
-          max = Math.max(max, Math.abs(f));
+          max = Math.max(max, (f - 128) / 128);
```

Comment:
Could you explain more about this logic?

Reply:
This math normalizes the values. Byte time domain data is stored 0-255 where 0...128...256 would be equivalent to -1...0...1.